cairo/src/utils.rs

```diff
@@ -4,6 +4,13 @@ use crate::error::Error;
 use std::ffi::CStr;
 use std::fmt;
 
+// rustdoc-stripper-ignore-next
+/// Resets all static data within cairo to its original state (i.e. identical to the state at program
+/// invocation). For example, all caches within cairo will be flushed empty.
+///
+/// # Safety
+/// It is only safe to call this function when there are no active cairo objects remaining (the
+/// appropriate destroy functions have been called as necessary).
```

Comment:
There are other `destroy` functions? Or do you mean the objects need to be dropped?

Also I assume this must not be called simultaneously from multiple threads?

Reply:
Got too immersed in the Cairo C docs :sweat_smile: 

> Also I assume this must not be called simultaneously from multiple threads?

The docs don't mention anything about this, but I looked at the source and it seems to use a mutex, I think it's safe to expose this detail in our documentation since just not using that mutex anymore would be a breaking change. We could just be more overly restrictive and tell people to not do this though